plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -225,6 +231,71 @@ public BigQueryConfig setQueryResultsCacheEnabled(boolean queryResultsCacheEnabl
         return this;
     }
 
+    public int getRpcInitialChannelCount()
+    {
+        return rpcInitialChannelCount;
+    }
+
+    @ConfigHidden
+    @Config("bigquery.rpc-initial-channel-count")
+    public BigQueryConfig setRpcInitialChannelCount(int rpcInitialChannelCount)
+    {
+        this.rpcInitialChannelCount = rpcInitialChannelCount;
+        return this;
+    }
+
+    public int getMinRpcPerChannel()
+    {
+        return minRpcPerChannel;
+    }
+
+    @ConfigHidden
+    @Config("bigquery.min-rpc-per-channel")
+    public BigQueryConfig setMinRpcPerChannel(int minRpcPerChannel)
+    {
+        this.minRpcPerChannel = minRpcPerChannel;
+        return this;
+    }
+
+    public int getMaxRpcPerChannel()
+    {
+        return maxRpcPerChannel;
+    }
+
+    @ConfigHidden
+    @Config("bigquery.max-rpc-per-channel")
+    public BigQueryConfig setMaxRpcPerChannel(int maxRpcPerChannel)
+    {
+        this.maxRpcPerChannel = maxRpcPerChannel;
+        return this;
+    }
+
+    public int getRpcMaxChannelCount()
+    {
+        return rpcMaxChannelCount;
+    }
+
+    @ConfigHidden
+    @Config("bigquery.rpc-max-channel-count")
+    public BigQueryConfig setRpcMaxChannelCount(int rpcMaxChannelCount)
+    {
+        this.rpcMaxChannelCount = rpcMaxChannelCount;
+        return this;
+    }
+
+    public int getRpcMinChannelCount()
+    {
+        return rpcMinChannelCount;
+    }
+
+    @ConfigHidden
+    @Config("bigquery.rpc-min-channel-count")
```

Comment:
How about renaming like below? I feel the current naming are little confusing as they're too similar. 
* bigquery.rpc-initial-channel-count → bigquery.channel-pool.initial-size
* bigquery.min-rpc-per-channel → bigquery.channel-pool.min-rpc-size-per-channel
* bigquery.max-rpc-per-channel → bigquery.channel-pool.max-rpc-size-per-channel
* bigquery.rpc-max-channel-count → bigquery.channel-pool.max-size
* bigquery.rpc-min-channel-count → bigquery.channel-pool.min-size